src/App.tsx

```diff
@@ -43,11 +37,21 @@ function App() {
     setIsNewCardDisplayed(false);
   };
 
-  const handleDeleteButtonClick = (id: number) => {
-    const newCardsArray = cardsArray.filter((_, index) => index != id)
+  const handleDeleteButtonClick = (id: string) => {
+    const newCardsArray = cardsArray.filter((card) => card.id != id)
     setCardsArray(newCardsArray)
   }
 
+  
+  useEffect(() => {
+    importCards().then((importedCardsArray) => {
```

Comment:
**nitpick:** Let's name it more meaningful - like `fetchCards`, `getCards` or something similar. Import is rather reserved keyword for importing code parts across the codebase